lib/mayaUsd/ufe/Global.cpp

```diff
@@ -107,6 +108,10 @@ MCallbackId gExitingCbId = 0;
 // Subject singleton for observation of all USD stages.
 MayaUsd::ufe::MayaStagesSubject::RefPtr g_StagesSubject;
 
+void mayaStartWaitCursor() { MGlobal::executeCommand("waitCursor -state 1"); }
```

Comment:
This seems like something that should be in mayaUsd/Utils and not Ufe, as this isn't manipulating or using anything Ufe and I think it is going to be useful for the rest of MayaUsd.

Reply:
Other code can use the usdUfe API, not this implementation. Particularly since the code is usdUfe manages the recursivity count and provide a utility WaitCursor class.

Reply:
mayaUsd has access to usdUfe but probably not the other way around. 

I also noticed that you've removed the struct in MayaUsdContextOps.

Reply:
This particular code cannot be in UsdUfe since it uses Maya. UsdUfe does not (and cannot) depend on Maya.